internal/filter/chain.go

```diff
@@ -41,37 +41,25 @@ func Chain(filters ...transport.Filter) transport.Filter {
 // filterChain combines a series of filters into a single Filter.
 type chain []transport.Filter
 
-func (c chain) Call(ctx context.Context, request *transport.Request, out transport.Outbound) (*transport.Response, error) {
+func (c chain) Send(ctx context.Context, request *transport.Request, sender transport.RequestSender) (*transport.Response, error) {
```

Comment:
`SendChain` implies that there's always a chain or stack of calls to go
through. For the rest of the system `RequestSender` is really just "this will
actually make the request and return the response."